src/model/image-environment-factory.ts

```
@@ -82,17 +82,11 @@ class ImageEnvironmentFactory {
-    if (parameters.providerStrategy === 'local-docker') {
-      for (const element of additionalVariables) {
-        if (!environmentVariables.some((x) => element?.name === x?.name)) {
-          environmentVariables.push(element);
-        }
-      }
-      for (const variable of environmentVariables) {
-        if (!environmentVariables.some((x) => variable?.name === x?.name)) {
-          environmentVariables = environmentVariables.filter((x) => x !== variable);
-        }
-      }
+    // Always merge additional variables (e.g., secrets/env from Cloud Runner) uniquely by name
```

ESLint

[error] Expected line before comment.

(lines-around-comment)